src/plugins/chart_expressions/expression_heatmap/public/expression_renderers/heatmap_renderer.tsx

```diff
@@ -79,24 +80,26 @@ export const heatmapRenderer: (
 
     render(
       <KibanaThemeProvider theme$={core.theme.theme$}>
-        <div className="heatmap-container" data-test-subj="heatmapChart">
-          <HeatmapComponent
-            {...config}
-            onClickValue={onClickValue}
-            onSelectRange={onSelectRange}
-            timeZone={timeZone}
-            datatableUtilities={getDatatableUtilities()}
-            formatFactory={getFormatService().deserialize}
-            chartsThemeService={plugins.charts.theme}
-            paletteService={getPaletteService()}
-            renderComplete={renderComplete}
-            uiState={handlers.uiState as PersistedState}
-            interactive={isInteractive()}
-            chartsActiveCursorService={plugins.charts.activeCursor}
-            syncTooltips={config.syncTooltips}
-            syncCursor={config.syncCursor}
-          />
-        </div>
+        <I18nProvider>
```

Comment:
non related to the main issue, but just saw it in the console and decided to fix